pkg/plugins/backendplugin/coreplugin/core_plugin.go

```diff
@@ -143,10 +144,10 @@ func (cp *corePlugin) ValidateAdmission(ctx context.Context, req *backend.Admiss
 	return nil, plugins.ErrMethodNotImplemented
 }
 
-func (cp *corePlugin) ConvertObject(ctx context.Context, req *backend.ConversionRequest) (*backend.ConversionResponse, error) {
+func (cp *corePlugin) ConvertObjects(ctx context.Context, req *backend.ConversionRequest) (*backend.ConversionResponse, error) {
 	if cp.AdmissionHandler != nil {
```

Comment:
this check is now different, it should be `if cp.ConversionHandler != nil`

Reply:
thanks! it looks like i missed a couple. updated in https://github.com/grafana/grafana/pull/92011